tests/launch-fastify.test.js

```diff
@@ -347,6 +347,94 @@ test('Test custom serializers', t => {
     await fastifyInstance.close()
   })
 
+  t.test('fields values - with custom properties on response log', async assert => {
+    assert.plan(18)
+    const stream = split(JSON.parse)
+
+    stream.once('data', () => {
+      stream.once('data', line => {
+        assert.equal(line.reqId, '34')
+        assert.equal(line.level, 10)
+        assert.notOk(line.req)
+        assert.strictSame(line.http, {
+          request: {
+            method: 'POST',
+            userAgent: { original: 'lightMyRequest' },
+          },
+        })
+        assert.strictSame(line.url, { path: '/items/my-item', params: { itemId: 'my-item' } })
+        assert.strictSame(line.host, { hostname: 'testHost', forwardedHostame: 'testForwardedHost', ip: 'testIp' })
+
+        stream.once('data', secondLine => {
```

Comment:
This is the stream of the log of the POST request, the one after this is the one of the GET request. But what is the one before this (from line 355 to line 366)?

Reply:
At line 355 it's the incoming request log, line 368 the request completed log for the POST request.

Here the incoming request for the GET and at line 387 the request completed log

Reply:
"line 368 the request completed log for the POST request." 
"Here the incoming request for the GET"
Here is exactly line 368.
So, line 355 is the incoming request log for the POST. Here (line 368) we have the request completed log for POST (in fact, they have the same reqId). 
Line 388 (`stream.once('data', thirdLine => {`) we have the request completed log for the GET (in fact, we have both request and response in the log).
Is it missing the incoming request log for the GET request?
Maybe it's not useful for the testing feature. It could make it clearer to read,  though

Reply:
>Is it missing the incoming request log for the GET request?
It's not missing, it's simply skipped at line 387

Reply:
Rename the stream function params as suggested and added another reqId assertion in log. So it should be easier to understand the request we assert